x-pack/packages/kbn-elastic-assistant-common/impl/schemas/actions_connector/post_actions_connector_execute_route.schema.yaml

```diff
@@ -0,0 +1,146 @@
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - params
+                - llmType
+              properties:
+                conversationId:
+                  $ref: '../conversations/common_attributes.schema.yaml#/components/schemas/UUID'
+                params:
+                  $ref: '#/components/schemas/ConnectorExecutionParams'
+                alertsIndexPattern:
+                  type: string
+                allow:
+                  type: array
+                  items:
+                    type: string
+                allowReplacement:
+                  type: array
+                  items:
+                    type: string
+                isEnabledKnowledgeBase:
+                  type: boolean
+                isEnabledRAGAlerts:
+                  type: boolean
+                replacements:
+                  type: object
+                  additionalProperties: true
+                size:
+                  type: number
+                llmType:
+                  type: string
+                  enum:
+                    - bedrock
+                    - openai
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: string
+                  connector_id:
+                    type: string
+                  replacements:
+                    type: object
+                    additionalProperties: true
+                  status:
+                    type: string
+                  trace_data:
+                    type: object
+                    description: Trace Data
+                    properties:
+                      transactionId:
+                        type: string
+                        description: Could be any string, not necessarily a UUID
+                      traceId:
+                        type: string
+                        description: Could be any string, not necessarily a UUID
+        '400':
+          description: Generic Error
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  statusCode:
+                    type: number
+                  error:
+                    type: string
+                  message:
+                    type: string
+
+components:
+  schemas:
+    RawMessageData:
+      type: object
+      additionalProperties: true
+
+    ConnectorExecutionParams:
+      type: object
+      description: AI assistant connector execution params.
+      required:
+        - subActionParams
+        - subAction
+      properties:
+        subActionParams:
+          type: object
+          required:
+            - messages
+          properties:
+            messages:
+              type: array
```

Comment:
Consider changing this from an `array` to just `message`, because it appears that we only send the last question asked by the user.

This feels like a good change, but now the API implies the server will do something with the previous messages, but the server now retrieves the previous messages from storage. Thus we should not send an array of messages anymore.